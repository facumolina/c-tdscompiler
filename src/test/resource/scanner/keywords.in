/* Keywords */
class A
if (b)
int x;
float f;
extern
return a;

/* Incorrect keywords */
classA
whiletrue
if(b)
intx;
breakcontinue;