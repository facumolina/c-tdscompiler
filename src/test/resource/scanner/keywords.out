COMMENT
CLASS
ID
IF
L_PAREN
ID
R_PAREN
INT
ID
SEMI_COLON
FLOAT
ID
SEMI_COLON
EXTERN
RETURN
ID
SEMI_COLON
COMMENT
ID
ID
IF
L_PAREN
ID
R_PAREN
ID
SEMI_COLON
ID
SEMI_COLON
